layers/newrelic/README.md

```diff
@@ -0,0 +1,56 @@
+# Bref New Relic Extension
+
+This is a Dockerfile to help add New Relic's PHP Agent to the Bref runtime so monitoring your applications on Lambda is possible. 
+
+## Install and configure
+
+```cli
+composer require bref/extra-php-extensions
+```
+
+```yaml
+# serverless.yml
+service: app
+
+provider:
+    name: aws
+    region: us-east-1
+    runtime: provided
+
+plugins:
+    - ./vendor/bref/bref
+    - ./vendor/bref/extra-php-extensions # <----- Add the extra Serverless plugin
+
+functions:
+    console:
+        handler: bin/console
+        layers:
+            - ${bref:layer.php-74}
+            - ${bref:extra.newrelic-php-74}
+            - ${bref:layer.console}
+```
+
+### Extra configuration files
+
+The configuration in the PHP.ini for New Relic is purposely using Environment Variables. In your Serverless config you can simple add them manually or store them in something like KMS and reference them as required if the setting is a token/key/password.
```

Comment:
Does a php.ini file really read environment variables like this?

Hm.. I still thing you should specify that in your app. You can still use environment variables if you want to.

Reply:
Can you provide an example. I elaborated above on what these are being used for but the ENV VARS seemed a scalable way to provide unique configuration per Lambda and control the settings when deploying quickly and easily. Totally open to switching, just wondering what the preferred approach is